.github/actions/security-audit/scripts/sarif-to-json.py

```diff
@@ -0,0 +1,190 @@
+import argparse
+import json
+import os
+import re
+from collections import defaultdict
+from typing import Dict, NamedTuple
+
+class BaseInfo(NamedTuple):
+    """
+    A named tuple for base information
+
+    Named tuples assign meaning to each position in a tuple and allow for more readable, self-documenting code
+    They add the ability to access fields by name instead of position index
+    """
+    location: Dict
```

Comment:
Try to replace using `Dict` from the typing to just `dict`? So we don't need to import it. I think it should be possible?
```suggestion
    location: dict
```